src/west/app/config.py

```diff
@@ -64,7 +69,7 @@
 
 CONFIG_EPILOG = '''\
 If the configuration file to use is not set, reads use all three in
-precedence order, and writes use the local file.'''
+precedence order, and writes (including appends) use the local file.'''
```

Comment:
I'm afraid you just discovered a documentation bug (or worse). The `west config` works just fine outside of any workspace - when there is no "local file". What happens then?

Reply:
You get this:
`ValueError: ConfigFile.LOCAL: file not found; retry in a workspace or set WEST_CONFIG_LOCAL`
So, because you can actually set `WEST_CONFIG_LOCAL` I don't think this is really a bug?

Reply:
Thanks, I should have tried. It's a small documentation... gap, not worth filling because the error message is good enough and this help message is already long enough.